templates/ads/dfp-amp--dfp-for-amp-flying-carpet.html.twig

```diff
@@ -1,7 +1,18 @@
 {{ attach_library('infinite_amp/amp.fx-flying-carpet') }}
 
+{% if rtc_config %}
+    {% set vendors = rtc_config.vendors %}
+    {% set vendors = vendors|merge({aps: {"PUB_ID": "3690","PARAMS":{"amp":"1"}}}) %}
```

Comment:
We should look at storing `PUB_ID` in a config somewhere. I will create a ticket.

Reply:
https://jira.burda.com/browse/INREL-6512